### DISCUSSION TITLE
[Project beta] Project README seems have word limit

### DISCUSSION THREAD

**Discussion by yuhan** [community/community#12522] [Projects and Issues]
How long can the README be? I was drafting a long README which includes detailed description for different fields but it seems like there is a word limit that I can't type further (after about 160 words?

Besides, what's the recommended way to document different fields and views? Can we have descriptions for "Field"s and `View"s?

**Comment by mfalconi-perle** [comment]
If you open the setting tab next to where the readme tab open, you will find a secondary editor that allows you to edit the project beta readme
![image](https://user-images.githubusercontent.com/96071862/166472572-938d8f24-4f9b-4880-8ddd-a7a61f7a9a11.png)
![image](https://user-images.githubusercontent.com/96071862/166473103-3ef89345-6058-48a0-89f3-6a594494b222.png)

**Reply by mfalconi-perle** [reply]
this view appears to not limit the amount of characters you can input

**Comment by dabrunetti** [comment]
There is a 1024 character limit on the readme
